diff --git a/node_modules/discord-api-types/payloads/v9/_interactions/responses.d.ts b/node_modules/discord-api-types/payloads/v9/_interactions/responses.d.ts
index 76aebca..87411ec 100644
--- a/node_modules/discord-api-types/payloads/v9/_interactions/responses.d.ts
+++ b/node_modules/discord-api-types/payloads/v9/_interactions/responses.d.ts
@@ -3,7 +3,7 @@ import { RESTPostAPIWebhookWithTokenJSONBody } from '../../../v9';
 /**
  * https://discord.com/developers/docs/interactions/slash-commands#interaction-object-interaction-request-type
  */
-export declare const enum InteractionType {
+export enum InteractionType {
     Ping = 1,
     ApplicationCommand = 2,
     MessageComponent = 3
@@ -33,7 +33,7 @@ export interface APIInteractionResponseUpdateMessage {
 /**
  * https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-callback-type
  */
-export declare const enum InteractionResponseType {
+export enum InteractionResponseType {
     /**
      * ACK a `Ping`
      */

diff --git a/node_modules/discord-api-types/payloads/v9/channel.d.ts b/node_modules/discord-api-types/payloads/v9/channel.d.ts
index cf215cf..f3e0531 100644
--- a/node_modules/discord-api-types/payloads/v9/channel.d.ts
+++ b/node_modules/discord-api-types/payloads/v9/channel.d.ts
@@ -468,7 +468,7 @@ export declare const enum MessageActivityType {
 /**
  * https://discord.com/developers/docs/resources/channel#message-object-message-flags
  */
-export declare const enum MessageFlags {
+export enum MessageFlags {
     /**
      * This message has been published to subscribed channels (via Channel Following)
      */
